src/core/respec-global.js

```diff
@@ -33,6 +47,7 @@ class ReSpec {
 export function init() {
   const respec = new ReSpec();
   Object.defineProperty(document, "respec", { value: respec });
+  document.dispatchEvent(new CustomEvent("respec-start"));
```

Comment:
Can we use the existing `<script onload>` for this?

Reply:
Should we fire this at all... who is expected to use this?

Reply:
`onload` can get fired before the code that adds `document.respec` is run.

> who is expected to use this

`respecDocWriter` can use it to start listening to error/warning events, similar to how we add a console message listener. Tests are also using it.

It's a guarantee that respec has started processing and is emitted as soon as respec starts (a little bit after DOMReady).

Reply:
maybe we should call it "private-respec-start" or something - just in case people find it and are tempted to use it.

Reply:
Let me add it when I use it in respec2html. Might figure out a way to not need it.

Reply:
What's wrong if people use it though?

Reply:
We have multiple mechanism in place already for people to use, so I'm worried about the maintainability of having yet another that could cause issues down the road. Want to avoid another `w3c-common` situation.

Reply:
Renamed to `__private_respec_start__`